samples/Common.c

```diff
@@ -554,10 +554,16 @@ STATUS createSampleStreamingSession(PSampleConfiguration pSampleConfiguration, P
 
     ATOMIC_STORE_BOOL(&pSampleStreamingSession->terminateFlag, FALSE);
     ATOMIC_STORE_BOOL(&pSampleStreamingSession->candidateGatheringDone, FALSE);
-
+    MEMSET(&pSampleStreamingSession->twccMetadata, 0x00, SIZEOF(TwccMetadata));
     pSampleStreamingSession->peerConnectionMetrics.peerConnectionStats.peerConnectionStartTime = GETTIME() / HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
     // Flag to enable SDK to calculate selected ice server, local, remote and candidate pair stats.
     pSampleConfiguration->enableIceStats = FALSE;
+    pSampleConfiguration->enableTwcc = TRUE;
```

Comment:
I see in that case doesn't it make sense to have this toggle inside `createSampleConfiguration` instead of this function since the enableTwcc flag is part of the sample configuration??

And in that case we should have a 1-line comment above where we set it that this toggle is used to enable/disable twcc with a link to a page that explains what twcc is (ideally a rfc or something we have reasonably certainty the url won't change).  I suppose I have the same question about the `enableIceStats` why don't we set that inside `createSampleConfiguration` ?

Reply:
Moved both. Added TWCC related details in the README.

Reply:
thank you!